app-admin/kernel-tools/spec

```diff
@@ -1,8 +1,12 @@
-VER=6.9.7
+VER=6.10.0
 #RC=
 # Use this for RC releases.
+#
+# To RFC: Increment by 0.1 per RC release, i.e., RC3 = 0.3.
+#REL=0.${RC}
+REL=1
 #SRCS="tbl::https://git.kernel.org/torvalds/t/linux-${VER%%.0}-rc${RC}.tar.gz"
```

Comment:
gz? xz?

Reply:
RC tarballs are only available in .tar.gz.